src/inference/src/model_reader.cpp

```diff
@@ -118,6 +118,10 @@ std::shared_ptr<ov::Model> read_model(const std::string& modelPath,
     std::string model_path = modelPath;
 #endif
 
+#if !defined(__EMSCRIPTEN__) && !defined(__ANDROID__)
+    OPENVINO_ASSERT(!fs::is_symlink(model_path), "Cannot read model\"" + modelPath + "\". The path is a symlink!");
+#endif
```

Comment:
Hi @zhaixuejun1993, huggingface models are saved as symbolic links in a local cache. After this fix, we can't run any huggingface model.
Please check this one example from [huggingface/OpenVINO](https://huggingface.co/OpenVINO/stable-diffusion-v1-5-fp16-ov)

```python
from optimum.intel.openvino import OVStableDiffusionPipeline

model_id = "OpenVINO/stable-diffusion-v1-5-fp16-ov"
pipeline = OVStableDiffusionPipeline.from_pretrained(model_id)

prompt = "sailing ship in storm by Rembrandt"
images = pipeline(prompt).images
```